examples/llava/clip.h

```diff
@@ -36,7 +40,7 @@ struct clip_image_f32_batch {
     size_t size;
 };
 
-CLIP_API struct clip_ctx * clip_model_load    (const char * fname, int verbosity);
+CLIP_API struct clip_ctx * clip_model_load    (const char * fname, int verbosity, struct load_image_size * load_image_size = nullptr);
```

Comment:
Don't use default arguments in the C API:

```suggestion
CLIP_API struct clip_ctx * clip_model_load    (const char * fname, int verbosity, struct load_image_size * load_image_size);
```

Also, passing the image size to `clip_model_load` seems weird from API point of view. Maybe use some worst-case image size instead?